app/views/maps/map.html.erb

```diff
@@ -38,9 +38,13 @@
         '<div class="popup-preview">' +
           '<div class="organization"><%= user.organization %></div>' +
           '<div class="user-container">' +
+            '<%= link_to user_path(id: user.id) do %> ' +
             '<div class="name"><%= user.first_name %> <%= user.last_name %></div>' +
-              // TODO: Use user's image
-              '<div class="user-image" style="background-image:url(\'https://scontent-sjc2-1.xx.fbcdn.net/v/t1.0-1/p64x64/13754547_10206918263334703_2191355668978129752_n.jpg?oh=d7c8d20514325912d1898ea9f9982ad8&oe=58B58FF7\')"></div>' +
+            '<% end %>' +
+            '<% if user.profile_pic? %> ' +
```

Comment:
you should make a helper function that spits out  this stuff and use that everywhere there needs to be a profile pic